nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java

```diff
@@ -139,6 +145,20 @@ public class UnpackContent extends AbstractProcessor {
                     PackageFormat.FLOWFILE_STREAM_FORMAT_V2.toString(), PackageFormat.FLOWFILE_TAR_FORMAT.toString())
             .defaultValue(PackageFormat.AUTO_DETECT_FORMAT.toString())
             .build();
+    public static final PropertyDescriptor ZIP_FILENAMES_ENCODING = new PropertyDescriptor.Builder()
+            .name("zip-filenames-encoding")
```

Comment:
Consider just 'filename-charset' for (name) and 'Filename Character Set' for Display name.

This only applies when it is zip format selection so no need to add 'zip' to the name as the context for use is already established.

Reply:
Actually, using `Filename Character Set` for both the name and display name would be optimal.